README.md

```diff
@@ -15,6 +15,24 @@ Just install gradle and run:
 
     gradle run
 
+### Run with multiple machines
+- Connect machines to a common network
+- Find the ip of the server/host machine (for example `192.168.1.123`)
+- Find a port which can be used for JADE communication (for example `5555`)
+
+For server/host machine
+
+    gradle run --args="-isHost 192.168.1.123 -port 5555 -stage1 -stage2 ... -noTK"
+
+For client machines
+
+    gradle run --args="-host 192.168.1.123 -localPort 5555 -stage3 -noTK"
+    
+For the client which executes gradle command later than every other machine
+
+    gradle run --args="-host 192.168.1.123 -localPort 5555 -stageN"
```

Comment:
Also here

Reply:
Could you test it with multiple machines?